seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -135,34 +137,31 @@ def test_get_workspace_acl(self, mock_time, mock_logger):
         mock_time.time.return_value = AUTH_EXTRA_DATA['auth_time'] + 10
 
         url = '{}api/workspaces/my-seqr-billing/my-seqr-workspace/acl'.format(TEST_TERRA_API_ROOT_URL)
-        responses.add(responses.GET, url, status = 200, body = '{"acl": {"test1@test1.com": {"accessLevel": "OWNER","canCompute": true,"canShare": true,"pending": false},"sf-seqr@my-seqr.iam.gserviceaccount.com": {"accessLevel": "OWNER","canCompute": true,"canShare": true,"pending": false},"test2@test2.org": {"accessLevel": "OWNER","canCompute": true,"canShare": true,"pending": false},"test3@test3.com": {"accessLevel": "READER","canCompute": false,"canShare": false,"pending": false}}}')
+        responses.add(responses.GET, url, status=200, body='{"acl": {"test1@test1.com": {"accessLevel": "OWNER","canCompute": true,"canShare": true,"pending": false},"sf-seqr@my-seqr.iam.gserviceaccount.com": {"accessLevel": "OWNER","canCompute": true,"canShare": true,"pending": false},"test2@test2.org": {"accessLevel": "OWNER","canCompute": true,"canShare": true,"pending": false},"test3@test3.com": {"accessLevel": "READER","canCompute": false,"canShare": false,"pending": false}}}')
         acl = user_get_workspace_acl(user, 'my-seqr-billing', 'my-seqr-workspace')
         self.assertIn('test3@test3.com', acl.keys())
         mock_logger.info.assert_called_with(
             'GET https://terra.api/api/workspaces/my-seqr-billing/my-seqr-workspace/acl 200 425 test_user')
         self.assertEqual(len(mock_logger.method_calls), 1)
 
         mock_logger.reset_mock()
-        responses.replace(responses.GET, url, status = 401)
+        responses.replace(responses.GET, url, status=401)
         with self.assertRaises(TerraAPIException) as ec:
             _ = user_get_workspace_acl(user, 'my-seqr-billing', 'my-seqr-workspace')
         self.assertEqual(str(ec.exception),
             'Error: called Terra API: GET /api/workspaces/my-seqr-billing/my-seqr-workspace/acl got status: 401 with a reason: Unauthorized')
         self.assertEqual(ec.exception.status_code, 401)
-        mock_logger.error.assert_called_with(
-            'GET https://terra.api/api/workspaces/my-seqr-billing/my-seqr-workspace/acl 401 0 test_user')
-        self.assertEqual(len(mock_logger.method_calls), 1)
```

Comment:
Same as above, I don't see any changes to `user_get_workspace_acl ` so why is the test changing?